app/models/beta/search/facet_filter_statistic_collection.rb

```diff
@@ -29,8 +29,7 @@ def facet_filter_applicable?(facet_filter_statistic)
           facet_filter_statistic.facet_count,
         )
 
-        facet_filter_statistic.facet_classification_statistics.many? &&
```

Comment:
If we require all facet filters to have multiple classifications then we can't see ones that we've clicked to filter by the classification in the UI and cause an error